src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,34 @@
+.catalog-container {
+  display: grid;
+  margin: 0 auto;
+  padding-top: 50px;
+  padding-bottom: 72px;
+  row-gap: 46px;
+  column-gap: $gap;
+  justify-content: center;
+
+  grid-template-columns: repeat(var(--card-count), 200px);
+  max-width: $card-width;
+  --card-count: 1;
+}
```

Comment:
`--card-count` is a CSS variable, it's better to use SCSS variable e.g. `$card-count` instead